### PR TITLE
Rhel5 build fixes

### PR DESCRIPTION
**Pull request by bretambrose**
By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: source/hpack.c [hunk 1]
```diff
@@ -1261,9 +1261,7 @@ int aws_hpack_decode(
                 goto handle_complete;
             } break;
 
-            default: {
```
Comment (TingDaoK) [inline source/hpack.c:1264]: When I run Format Document at windows machine, this will happen. Undo this change will pass the Lint for format...

Comment (bretambrose) [reply source/hpack.c:1264]: Yeah, for some reason, the clang format on my machine is slightly out of sync with what's in CI
